level1c4pps/tests/test_seviri2pps.py

```diff
@@ -613,6 +613,22 @@ def test_get_calibration(self, platform, timestamp, expected):
         coefs = calib.get_calibration(platform=platform, time=timestamp)
         self._assert_coefs_close(coefs, expected)
 
+
+    def test_get_calibration_ir_no_file(self):
+        """Test get ir calibration with mising json file."""
+        with pytest.raises(FileNotFoundError):
+            coefs = calib.get_calibration(
+                platform="MSG2",
+                time=dt.datetime(2007, 6, 5, 0, 0),
+                path_to_external_ir_calibration=".")
+
+    def test_get_calibration_ir(self):
+        """Test get ir calibration with mising json file."""
```

Comment:
is the file actually missing as the doc string suggests? 
In this case what is the difference to the test above?
If doc sting is not correct, I guess we should add an assert on the obtained coeffs. Maybe it makes sense to add the date to the return value of the functions of concern and assure that we get coefficients for expected date.